tests/test_cli.py

```diff
@@ -192,6 +192,45 @@ def test_command_exits_with_positive_return_code_when_issues_are_found(
             result = runner.invoke(cli.main, ["scan-local-repo", "."])
         self.assertGreater(result.exit_code, 0)
 
+    @mock.patch("tartufo.util.write_outputs", new=mock.MagicMock())
+    @mock.patch("tartufo.util.echo_result", new=mock.MagicMock())
+    @mock.patch("tartufo.commands.scan_local_repo.GitRepoScanner")
+    def test_command_exits_with_zero_return_code_when_no_issues_are_found(
+        self, mock_scanner: mock.MagicMock
+    ):
+        mock_scanner.return_value.issues = []
+        runner = CliRunner()
+        with runner.isolated_filesystem():
+            result = runner.invoke(cli.main, ["scan-local-repo", "."])
+        self.assertEqual(result.exit_code, 0)
+
+    @mock.patch("tartufo.util.write_outputs", new=mock.MagicMock())
+    @mock.patch("tartufo.util.echo_result", new=mock.MagicMock())
+    @mock.patch("tartufo.commands.scan_local_repo.GitRepoScanner")
+    def test_command_raises_error_when_quiet_and_verbose_simultaneously(
+        self, mock_scanner: mock.MagicMock
+    ):
+        mock_scanner.return_value.issues = []
+        runner = CliRunner()
+        with runner.isolated_filesystem():
+            result = runner.invoke(cli.main, ["-q", "-v", "scan-local-repo", "."])
+        self.assertEqual(result.exit_code, 2)
+
+    @mock.patch("tartufo.util.write_outputs", new=mock.MagicMock())
+    @mock.patch("tartufo.util.echo_result", new=mock.MagicMock())
+    @mock.patch("tartufo.commands.scan_local_repo.GitRepoScanner")
+    def test_command_returns_with_zero_when_quiet_or_verbose_only(
+        self, mock_scanner: mock.MagicMock
+    ):
+        mock_scanner.return_value.issues = []
+        runner = CliRunner()
+        with runner.isolated_filesystem():
+            result = runner.invoke(cli.main, ["-q", "scan-local-repo", "."])
+        self.assertEqual(result.exit_code, 0)
+        with runner.isolated_filesystem():
+            result = runner.invoke(cli.main, ["-v", "scan-local-repo", "."])
+        self.assertEqual(result.exit_code, 0)
```

Comment:
This should be split out into two separate tests. I know it's repetitive, but that way we know exactly what's failing if somehow one of these breaks.

Reply:
Done